CHANGELOG.md

```diff
@@ -1,3 +1,75 @@
+## v0.6.1 - 2021-02-09
+
+This is a patch release which includes mainly bug fixes.
+
+> **NOTE**: Please read the updating guidelines [here](https://github.com/kinvolk/lokomotive/blob/v0.6.1/CHANGELOG.md#updating-from-v060).
+
+### Changes in v0.6.1
+
+#### Development
+
+- Velero: Add tolerations to Restic plugin ([#1348](https://github.com/kinvolk/lokomotive/pull/1348)).
+- Velero: Add e2e tests ([#1353](https://github.com/kinvolk/lokomotive/pull/1353)).
+- Update all Go dependencies ([#1358](https://github.com/kinvolk/lokomotive/pull/1358)).
+
+#### Terraform Provider Updates
+- Update Packet (Equinux Metal) terraform provider to 3.2.1 ([#1349](https://github.com/kinvolk/lokomotive/pull/1349)).
```

Comment:
```suggestion
- Update Packet (Equinux Metal) Terraform provider to 3.2.1 ([#1349](https://github.com/kinvolk/lokomotive/pull/1349)).
```

Reply:
Also, should we mention what bug does this fix, perhaps in other section?

Reply:
I think its not required. what do others think ? 
@iaguis @surajssd @knrt10

Reply:
I think, having this in the changelog is sufficient, as PR itself explains what it fixes.

Reply:
As a user who experienced `n2.xlarge.86` provisioning failures, it would be nice to get notified that this has been fixed IMO.

Reply:
rephrased